yuniql-cli/CommandLineService.cs

```diff
@@ -368,10 +368,64 @@ public int RunArchiveOption(ArchiveOption opts)
                 return OnException(ex, "Failed to execute archive function", opts.IsDebug);
             }
         }
+        public int RunConfigOption(ConfigOption opts)

```

Comment:
I understand this piece tries to determine where the value came from if its from CLI parameter, environment variable or just default. Thank you.